ql/src/java/org/apache/hadoop/hive/ql/exec/TaskFactory.java

```diff
@@ -108,7 +110,7 @@ public TaskTuple(Class<T> workClass, Class<? extends Task<T>> taskClass) {
         IndexMetadataChangeTask.class));
     taskvec.add(new TaskTuple<TezWork>(TezWork.class, TezTask.class));
     taskvec.add(new TaskTuple<SparkWork>(SparkWork.class, SparkTask.class));
-
+    taskvec.add(new TaskTuple<>(ReplDumpWork.class, ReplDumpTask.class));
```

Comment:
it picks up the corresponding class from the arguments, with java 1.8 specifying the above is redundant.